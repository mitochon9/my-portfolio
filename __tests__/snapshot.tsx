import renderer from "react-test-renderer";

import IndexPage from "../pages/index";

it("renders index unchanged", () => {
  const tree = renderer.create(<IndexPage />).toJSON();
  expect(tree).toMatchSnapshot();
});

