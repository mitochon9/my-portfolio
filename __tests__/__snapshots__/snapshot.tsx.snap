// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders index unchanged 1`] = `
<div
  className="flex flex-col min-h-screen"
>
  <header>
    <div
      className="border-b"
    >
      <div>
        Header
      </div>
    </div>
  </header>
  <main
    className="flex-1 bg-blue-100"
  >
    <div>
      <h2>
        Index
      </h2>
    </div>
  </main>
  <footer>
    <div
      className="border-t"
    >
      <div>
        Footer
      </div>
    </div>
  </footer>
</div>
`;
